akvo/rsr/dir/app/modules/project-page/components/Indicator/Indicator.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react'
+import { Empty, Skeleton } from 'antd'
+import { useDispatch } from 'react-redux'
+
+import Quantitative from './Quantitative'
+import Qualitative from './Qualitative'
+import PeriodApi from '../PeriodApi'
+
+import { getPeriodsByID } from '../../queries'
+import { setPeriods } from '../../../../features/results/resultSlice'
+import { append as appendFilter } from '../../../../features/periods/filterSlice'
+
+const Indicator = ({
+  id,
+  dimensionNames,
+  periods,
```

Comment:
here as well you can do `periods = []` so that if periods is null or undefined it will not break anything